### PR TITLE
[#mhv-48450] updates medications list and details print

### PR DESCRIPTION
**Pull request by KolbyVA**
## Summary
Updates the print functionality for my-health/medications application

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-48451

## Testing done
unit tests updated and passing

## Screenshots
Screen Shot 2023-09-19 at 11 50 59 AM
Screen Shot 2023-09-19 at 11 51 18 AM

## What areas of the site does it impact?
my/health/medications

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/medications/components/shared/PrintDownload.jsx [hunk 1]
```diff
@@ -1,57 +1,69 @@
...
+      className="link-button vads-u-margin-bottom--3"
+      onClick={download}
+    >
+      <i className="fas fa-download vads-u-margin-right--0p5" />
+      {list
+        ? 'Download your medication list as a PDF'
+        : 'Download your medication details as a PDF'}
+    </button>
+
+    // DROPDOWN PRINT/DOWNLOAD BUTTON
```
Comment (robertbylight) [inline src/applications/mhv/medications/components/shared/PrintDownload.jsx:33]: same as above

**Review by robertroddev — CHANGES_REQUESTED** [review]

File: src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx [hunk 2]
```diff
@@ -153,10 +153,22 @@ const VaPrescription = prescription => {
...
+                <div className="print-only">
+                  {entry.cmopNdcNumber ? (
+                    <img
+                      src={getImageUri(entry.cmopNdcNumber)}
+                      alt="Not Available"
+                      width="350"
+                      height="350"
+                    />
+                  ) : (
+                    <p className="vads-u-margin--0">Image not available</p>
```
Comment (robertbylight) [inline src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx:169]: I think this may be a duplicate from line 149?

Comment (robertbylight) [reply src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx:169]: Gotcha, yeah i missed that print-only class on my initial review. So then that makes me curious now as to why the alt is different on 151 from the print-only view?

**Review by robertroddev — APPROVED** [review]
Nice! good work! 👍

**Pull request merged**